### ISSUE TITLE
NavigationUiComponent needs multiple clicks on Desktop to restick the navigation flyout

### ISSUE THREAD

**Issue by 749** [SAP/spartacus#9688]
**Describe the bug**
In the Desktop view, when clicking back and forth between two of the Categories after each has been clicked once, you need two mouse clicks to make the Category stick open again.

**Tell us the version of Spartacus**
Noticed on:
- 2.1.1
Also exists on current Demo store: (probably core-3.0.0-next.6)?

**To Reproduce**
1. Go to https://spartacus-demo.eastus.cloudapp.azure.com/electronics-spa/en/USD/
2. Click on 'Brands' within the Category navigation
3. The 'Brands' category stays open after moving away the mouse from the menu
4. Click on 'Digital Cameras'
5. The 'Digital Cameras' category stays open after moving the mouse away from the menu
6. Click on 'Brands' again

**Expected behavior**
7. The 'Brands' category stays open after moving the mouse away from the menu

**Actual behavior**
7. None of the categories stay open when moving the mouse away
8. Click on 'Brands' again
9. The 'Brands' category stays open after moving the mouse away from the menu

**Screenshots**
- not screenshotable, behaviour is non static

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser Chromium
 - Version 86.0.4240.183

**Smartphone (please complete the following information):**
- not applicable

**Additional context**
The problem comes out of the NavigationUiComponent (projects/storefrontlib/src/cms-components/navigation/navigation/navigation-ui.component.ts), which does not remove the "is-opened" css class from sibling and sibling-children nodes.

**Comment by Xymmer** [comment]
Clicking should do nothing in our case
Might be something to do with tabbing
Thanks for the bug report!
@developpeurweb let us know if this is something that can be fixed by styling

**Comment by developpeurweb** [comment]
It could be related to tabbing since the spacebar is used for opening the menu and could be equivalent to clicking.

This cannot be fixed with styling. The author suggests the source of the problem (`navigation-ui.component.ts`) under the "Additional context" section at the end.

Changes to main navigation will happen at some point as we move into the Screen Reader support milestone, therefore I'm re-labelling this ticket so we can revisit this bug then.

Animated GIF attached, notice CSS class added on-click in the inspector's window on the left side.

![menu-is-open](https://user-images.githubusercontent.com/15113219/101212812-5e716600-3647-11eb-8d60-4d667b8251d7.gif)

**Comment by giancorderoortiz** [comment]
Please fix conflicts in pr(if any), and approve/reject.
Would like to test this fix soon.

**Comment by peebeebee** [comment]
If I understand correctly, this PRQ has been open for a few months now. 
What's the hold up?
